initfiles/bash/etc/init.d/install-init.in

```diff
@@ -300,3 +300,13 @@ fi
 if [ -d ${CONFIG_DIR} ]; then
     date > ${CONFIG_DIR}/installed
 fi
+
+if [ ! -d ${LOG_DIR} ]; then
+   mkdir -p ${LOG_DIR}
+   chown ${user}:${user} ${LOG_DIR}
+fi
+[ -e  ${LOG_DIR}/hpcc-init.debug ] && chown ${user}:${user} ${LOG_DIR}/hpcc-init.debug
+
+createRuntime
```

Comment:
Ming I think if you move this up to line 304, you can them remove the LOG_DIR block.  Im pretty sure it's redundant in the createRuntime function.  Other than that, this looks solid.  Tested it and it works fine.